### PR TITLE
Add fuel duty

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
We can't run a microsim off this yet, but it adds the logic needed for a household calculator.

### CODE REVIEW

File: openfisca_uk/variables/household/consumption/fuel.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    value_type = float
+    unit = "currency-GBP"
+
+    def formula(household, period, parameters):
+        return household("petrol_spending", period) / household(
+            "petrol_price", period
+        )
+
+
+class diesel_spending(Variable):
```
Comment (MaxGhenis) [inline openfisca_uk/variables/household/consumption/fuel.py:27]: `diesel_spending` and `diesel_litres` are flipped

File: openfisca_uk/variables/household/consumption/fuel.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+class petrol_litres(Variable):
+    label = "Petrol volume"
+    documentation = "Total litres of petrol bought"
+    entity = Household
+    definition_period = YEAR
+    value_type = float
+    unit = "currency-GBP"
+
+    def formula(household, period, parameters):
+        return household("petrol_spending", period) / household(
```
Comment (MaxGhenis) [inline openfisca_uk/variables/household/consumption/fuel.py:22]:
```suggestion
        spending = household("petrol_spending", period)
        price = household("petrol_price", period)
        return spending / price
```

File: openfisca_uk/variables/household/consumption/fuel.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+class petrol_spending(Variable):
+    label = "Petrol spending"
+    documentation = "Spending on petrol"
+    entity = Household
+    definition_period = YEAR
+    value_type = float
+    unit = "currency-GBP"
+
+
+class petrol_litres(Variable):
```
Comment (MaxGhenis) [inline openfisca_uk/variables/household/consumption/fuel.py:13]: could you split these variables into their own files?

File: openfisca_uk/parameters/consumption/fuel/prices/petrol.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+description: Average price of unleaded petrol per litre.
```
Comment (MaxGhenis) [inline openfisca_uk/parameters/consumption/fuel/prices/petrol.yaml]:
```suggestion
description: Average price of unleaded petrol per litre, including fuel duty.
```

Comment (MaxGhenis) [inline openfisca_uk/parameters/consumption/fuel/prices/petrol.yaml]: Could this cite the CPI instead? ONS mentions average fuel prices [here](https://www.ons.gov.uk/economy/inflationandpriceindices/bulletins/consumerpriceinflation/january2022) but I didn't see a dataset off hand.

If not, I think average prices could deserve its own folder within `parameters`, e.g. `parameters.average_prices.fuel.petrol`. When I first saw `prices` in the tree I thought it was some sort of set price.

File: openfisca_uk/variables/household/consumption/fuel.py [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+class diesel_litres(Variable):
+    label = "Diesel volume"
+    documentation = "Total litres of diesel bought"
+    entity = Household
+    definition_period = YEAR
+    value_type = float
+    unit = "currency-GBP"
+
+
+class petrol_price(Variable):
```
Comment (MaxGhenis) [inline openfisca_uk/variables/household/consumption/fuel.py:50]: do we need these?

Comment (nikhilwoodruff) [reply openfisca_uk/variables/household/consumption/fuel.py:50]: Yeah, I think it's useful to allow users to enter the price they paid; I've seen it in other calculators.

**Comment by nikhilwoodruff** [comment]
Superseded by #568

**Pull request closed without merging**